index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+​
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>ToDoリスト</h1>
+    ​
+    <div>
+      <label><input type="radio" checked="checked" />すべて</label>
+      <label><input type="radio" />完了</label>
+      <label><input type="radio" />作業中</label>
+      <table>
+        <thead>
+            <tr>
+              <th >ID</th>
```

Comment:
スペースが空いているので削除しておきましょう。
```suggestion
              <th>ID</th>
```